### PR TITLE
WRP-19326: Updated node modules' major versions

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Moonstone sampler has a dependency that needs a major update.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated `babel-loader` dependency to v9.
There were no breaking changes affecting our code https://github.com/babel/babel-loader/releases.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
I have also updated minor versions of `enact/docs-utils` and `enact/ui-test-utils` for Moonstone root.
For sampler, minimum required node version is v14.15.0 https://github.com/babel/babel-loader/releases/tag/v9.0.0. I have also updated a minor version of `enact/storybook-utils`.

### Links
[//]: # (Related issues, references)
WRP-19326

### Comments
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]

File: samples/sampler/package.json [hunk 1]
```diff
@@ -16,6 +16,9 @@
     "serve-qa": "start-storybook -p 8080 -c .qa-storybook --ci"
   },
   "license": "Apache-2.0",
+  "engines": {
```
Comment (daniel-stoian-lgp) [inline samples/sampler/package.json]: Only for the cli related modules we need to add this this

File: samples/sampler/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 The following is a curated list of changes in the Enact Sampler, newest changes on the top.
 
+## unreleased
+
+- Updated `babel-loader` version to `^9.1.2`.
```
Comment (daniel-stoian-lgp) [inline samples/sampler/CHANGELOG.md]: Interesting. I did not notice this changelog file for the moonstone sampler. It seems it hasn't been updated for 4 years. @seunghoh , do we stil need this file?

**Review by daniel-stoian-lgp — APPROVED** [review]

File: samples/sampler/CHANGELOG.md (hunk 2 above)
Comment (hong6316) [reply samples/sampler/CHANGELOG.md:7]: Please revert samples/sampler/CHANGELOG.md
(This change does not need to be written)

Comment (seunghoh) [reply samples/sampler/CHANGELOG.md]: @daniel-stoian-lgp hm.. let me think about it. Anyway, I don't think this change log doesn't need. We actually don't put the dependency change in the change log

File: samples/sampler/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
...
+- Updated the minimum version of Node to `14.15.0`.
```
Comment (hong6316) [inline samples/sampler/CHANGELOG.md]: I cannot find this change in samples/sampler/package.json but CHANGELOG.md described it.
Is that intentional?

Comment (stanca-pop-lgp) [reply samples/sampler/CHANGELOG.md]: Sorry, it was a change I undid and I forgot to delete this. Thank you!

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**